API/controller/mealController.js

```diff
@@ -0,0 +1,16 @@
+import MealService from "../services/MealsServices";
+
+const MealContoller = {
+  fetchAllMeals(req, res) {
+    const allMeals = MealService.fetchAllMeals();
```

Comment:
This is an unnecessary abstraction. The job of this controller is to fetch all the meals. The logic for fetching all the meals should be written in this controller, as opposed to what you have here where we are calling another method that fetches all the meals.